code/modules/power/tesla/coil.dm

```diff
@@ -5,6 +5,7 @@
 	icon_state = "coil"
 	anchored = 0
 	density = 1
+	use_power = 0
```

Comment:
These are already 0...? This is the default value for this var.

Reply:
Well, it wasn't working before. And they were using power!